WeakAurasTemplates/TriggerTemplates.lua

```diff
@@ -1164,6 +1254,17 @@ function WeakAuras.CreateTemplateView(frame)
       else
         newView.backButton:Show();
       end
+
+      -- batchButton
+      newView.batchButton:Hide();
```

Comment:
You already hide it at the start of the function.

Reply:
indeed, fixed for the "back" button too